pkg/karmadactl/join/join.go

```diff
@@ -168,6 +179,7 @@ func (j *CommandJoinOption) RunJoinCluster(controlPlaneRestConfig, clusterConfig
 		ClusterProvider:    j.ClusterProvider,
 		ClusterRegion:      j.ClusterRegion,
 		ClusterZone:        j.ClusterZone,
+		ClusterZones:       j.ClusterZones,
```

Comment:
Here we don't migrate the `zone` to `zones`, and expect the cluster validation would deny the case that they co-exist. Am I right?

Reply:
yes, but I think it would be a little better to check ahead of time, I will improve it.

Reply:
Yes. since the joining process is probably not atomic.
I think just a validation would be fine.